templates/page_header/page-header.tpl.php

```diff
@@ -6,8 +6,29 @@
  */
 ?>
 <div class="ecl-page-header">
-  <?php print render($breadcrumb); ?>
+  <?php print render($header_bottom); ?>
```

Comment:
This part is very strange. We are rendering a whole region for the sole purpose of rendering the breadcrumb. It sounds a bit overkill compared to the ECL implementation.